tasks/build_tags.py

```diff
@@ -369,3 +369,27 @@ def _compute_build_size(ctx, build_exclude=None, flavor=AgentFlavor.base):
 
     statinfo = os.stat('bin/agent/agent')
     return statinfo.st_size
+
+
+def compute_config_build_tags(targets="all", build_include=None, build_exclude=None, flavor=AgentFlavor.base.name):
+    flavor = AgentFlavor[flavor]
+
+    if targets == "all":
+        targets = build_tags[flavor].keys()
+    else:
+        targets = targets.split(",")
+        if not set(targets).issubset(build_tags[flavor]):
+            print("Must choose valid targets. Valid targets are:")
+            print(f'{", ".join(build_tags[flavor].keys())}')
+            return
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Could you use an exception instead ?